praw/settings.py

```diff
@@ -19,10 +19,7 @@
 import os
 import sys
 
-try:
-    import ConfigParser as configparser
-except ImportError:
-    import configparser  # NOQA pylint: disable=F0401
+from six.moves import configparser
```

Comment:
As long as flake8 doesn't complain then it's fine. I've been trying to organize imports a bit more (system-level, third-party, the local, each with a space between), but since there is not static analysis for that syntax I won't be picky on PRs.